[HW] 1072939

weic0813 opened the pull request:
因老師的投影片中對ex4之實作說明較不明確，同學們在實作時也耗費較多時間，故在README中增加較為詳細之敘述
增加內容:
●增加git log使用時機及如何配合使用
●解決完conflict後須使用continue完成先前已下達的merge/rebase指令

Signed-off-by: 1072939 <s1072939@mail.ncyu.edu.tw>

jrjang commented on README.md:
```diff
@@ -9,7 +9,13 @@ bash <(curl -s https://raw.githubusercontent.com/jrjang/ncyu-2021/ex4/scripts/ex
 2. 切到 ncyu-2021-ex4 資料夾下. 依照投影片描述做修改
 
-3. 完成並commit後, 在 ncyu-2021-ex4 資料夾下, 執行以下指令
+3. 修改前可透過 `git log` 指令確認當前 commit 資訊，已得知後續 `git rebase -i` 後續所需之 commit 編號
```
請先加上投影片上需要完成項目. 然後再加上你提供的提示

weic0813 replied:
要完成項目也是加在README中嗎？
還是commit message中？

Commit:
[HW] 1072939

 欲解決問題:
 因老師的投影片中對 ex4 之實作說明較不明確，同學們在實作時也耗費較多時間，故在 README 中增加較為詳細之敘述

 解決方法:
 於 README.md 增加:
 ●增加 git log 使用時機及如何配合使用
 ●解決完 conflict 後須使用 continue 完成先前已下達的 merge/rebase 指令

Signed-off-by: 1072939 <s1072939@mail.ncyu.edu.tw>
